base/array.jl

```diff
@@ -1868,8 +1874,12 @@ CartesianIndex(2, 1)
 """
 function findprev(testf::Function, A, start)
     i = start
-    while i >= first(keys(A))
+    f = first(keys(A))
+    i < f && return nothing
+    while true
         testf(A[i]) && return i
+        i == f && break
+        # prevind(A, l) can throw/underflow
```

Comment:
```suggestion
        # prevind(A, f) can throw/underflow
```

(marking this one too since you can now batch these line commits)